localnet/poktrolld/config/appgate_server_config.yaml

```diff
@@ -0,0 +1,8 @@
+# Whether the server should sign all incoming requests with its own ring (for applications)
+self_signing: true
+# The name of the key that will be used to sign relays
+signing_key: app1
+# The host and port that the appgate server will listen on
+listening_endpoint: http://localhost:42069
+# tcp://<host>:<port> to a full pocket node for reading data and listening for on-chain events
+query_node_url: tcp://127.0.0.1:36657
```

Comment:
DISCUSSION: What do you think of renaming this to `pocket_node_url` instead of `query_node_url`?

I'm not asking to change it, but curious to hear which you think is better in the long-term?

Reply:
These names actually reflect the original variable names used in the code. If I recall correctly, we chose these names to emphasize the distinction between `PocketNodeUrl`, which is used for sending transactions, and `QueryNodeUrl`, which functions solely as a querier.

While in the context of the `AppGate` server, there might not be a clear distinction between transactions and queries, making `pocket_node_url` a suitable choice, it would be prudent to seek input from @bryanchriswhite and @h5law before finalizing the name.

Reply:
Yeah `QueryNodeUrl` was chosen as it only queries the chain and the `NetworkNodeUrl`. I think keeping this or mabye clarifying the `NetworkNodeUrl` to be a little more verbose in what its used for would be ideal